R/ResultsDataModel.R

```diff
@@ -462,7 +451,8 @@ uploadResults <- function(connectionDetails = NULL,
         } else {
           DatabaseConnector::insertTable(
             connection = connection,
-            tableName = paste(env$schema, env$tableName, sep = "."),
+            tableName = env$tableName,
```

Comment:
i am thinking DatabaseConnector will handle the bulk load based on the system variables? i.e. we dont need to allow an overide in this package?

> 
>   bulkLoad = Sys.getenv("DATABASE_CONNECTOR_BULK_UPLOAD"),
>   useMppBulkLoad = Sys.getenv("USE_MPP_BULK_LOAD"),

Reply:
yes, this is correct DatabaseConnector should handle that. I'm not sure how much difference it makes with sqlite. When we properly test, document and support other platforms this would be useful to include in a vignette.

Reply:
Well - the major contribution of this PR is the use of SqlRender with translation. i.e. all SQL in DiagnosticExplorer should be in OHDSI Sql form. We are translating the OHDSI SQl to either SQL render or Postgres, but with this framework - we can support any db that is supported by SqlRender, as long as we can have a dbPool connection.

If this logic is true --then BULKLOAD would be useful for the MPP db's that is supported